templates/web/buckinghamshire/report/new/roads_message.html

```diff
@@ -1,25 +1,31 @@
 <div id="js-roads-responsibility" class="box-warning hidden">
-    <strong>Not maintained by Buckinghamshire Council</strong>
     <div id="js-not-council-road" class="hidden js-responsibility-message js-roads-bucks">
+        <strong>Not maintained by Buckinghamshire Council</strong>
         <p>The selected road is not maintained by Buckinghamshire Council;
         to find out who owns this land/road please contact the Land Registry at
         <a href="https://www.gov.uk/search-property-information-land-registry">https://www.gov.uk/search-property-information-land-registry</a>.
         </p>
     </div>
     <div id="js-not-council-road-other" class="hidden js-responsibility-message js-roads-bucks">
+        <strong>Not maintained by Buckinghamshire Council</strong>
         <p>The selected road is not maintained by Buckinghamshire Council,
         but by a neighbouring council. Please move your pin slightly, to the
         other side of the border.
         </p>
     </div>
     <div id="js-not-council-road-he" class="hidden js-responsibility-message js-roads-bucks">
+        <strong>Not maintained by Buckinghamshire Council</strong>
         <p>The selected road is not maintained by Buckinghamshire Council,
         but by National Highways. Please follow this link to
         <a class="js-update-coordinates" href="https://www.fixmystreet.com/report/new?latitude=[% latitude |html %]&amp;longitude=[% longitude | html %]">FixMyStreet</a> to continue reporting your issue here.
         </p>
     </div>
     <div id="js-not-a-road" class="hidden js-responsibility-message js-roads-bucks">
+        <strong>Not maintained by Buckinghamshire Council</strong>
         <p>The location you have selected doesn't appear to be on a road.</p>
         <p>Please select a road on which to make a report.</p>
     </div>
+    <div id="js-not-council-car-park" class="hidden js-responsibility-message">
+        <p>There is no Buckinghamshire maintained car park in the area selected.</p>
```

Comment:
There's no fixmystreet.com equivalent template bit for this, so you get a blank box:
![image](https://user-images.githubusercontent.com/154364/149305339-0bc36f29-f5e3-4e67-aa69-d3c3a044ddd9.png)